auto_nag/multinaggers.py

```diff
@@ -33,9 +33,9 @@ def get_args_parser(self):
         parser = argparse.ArgumentParser(description=self.description())
         parser.add_argument(
             "-d",
```

Comment:
Here we keep the short flag to `-d` (not consistent). I feel a better flag name could be `--production` with no short flag version (we don't want to make it easier to do a mistake).

Reply:
Agreed, I'll change "--no-dryrun" to "production" (I'll keep the "dryrun" name for the variable just so we don't have to modify everything everywhere).